src/sentry/web/urls.py

```diff
@@ -359,17 +341,32 @@ def init_all_applications():
             url='https://docs.sentry.io/hosted/api/', permanent=False),
         name='sentry-api-docs-redirect'
     ),
-    url(r'^api/$', generic_react_page_view, name='sentry-api'),
-    url(r'^api/[^0]+/', generic_react_page_view),
+    url(r'^api/$',
+        RedirectView.as_view(pattern_name="sentry-api", permanent=False),
+    name='sentry-api-old'),
+    url(r'^api/applications/$',
+        RedirectView.as_view(pattern_name="sentry-api-applications", permanent=False)),
+    url(r'^api/[^0]+/',
+        RedirectView.as_view(pattern_name="sentry-api-details", permanent=False),
+    ),
     url(r'^out/$', OutView.as_view()),
 
     url(r'^accept-transfer/$', react_page_view, name='sentry-accept-project-transfer'),
     # User settings use generic_react_page_view, while any view
     # acting on behalf of an organization should use react_page_view
-    url(r'^settings/account/security/$',
-        generic_react_page_view,
-        name='sentry-account-settings-security'),
+    url(r'^settings/account/$', generic_react_page_view, name="sentry-account-settings"),
+    url(r'^settings/account/$', generic_react_page_view, name="sentry-account-settings-appearance"),
```

Comment:
Yeah, the account "appearance" section was merged with account details, but I wanted to make sure the route name: `sentry-account-settings-appearance` (just in case something still references the route by name).

Do you know if there's a more concise way to give a route multiple route names?

Reply:
Not that I know of